app.js

```diff
@@ -133,6 +137,51 @@ async function renderPage(response, page) {
   }
 }
 
+
+// REVIEW FORM - DONT REMOVE ----------------------------
+app.post("/movies/:movieId/review", (request, response) => {
```

Comment:
I would make this into a normal API endpoint instead, i.e. one that accepts JSON and exists under /api.